Tools/CISupport/ews-build/steps.py

```diff
@@ -2656,7 +2656,7 @@ def buildCommandKwargs(self, warnings):
         # https://bugs.webkit.org/show_bug.cgi?id=239455: The timeout needs to be >20 min to
         # work around log output delays on slower machines.
         # https://bugs.webkit.org/show_bug.cgi?id=247506: Only applies to Xcode 12.x.
-        if self.getProperty('fullPlatform') == 'mac-bigsur':
+        if self.getProperty('fullPlatform') == 'mac-monterey':
```

Comment:
We'll need a follow up to remove this special case for the timeout, it only applied to Big Sur.